docker-compose.yml

```diff
@@ -77,8 +77,8 @@ services:
     networks:
       default:
         ipv4_address: $BITCOIN_IP
-  lnd:
-    container_name: lnd
+  lightning:
```

Comment:
@AaronDewes need to change the container name in some more places. lnd -> lightning
EDIT: fixed [911988f](https://github.com/runcitadel/core/pull/17/commits/911988ff37468b3eff4290ee2d1d42cdd690ed8f)

Reply:
Thanks!